addon/components/sessions-grid.js

```diff
@@ -46,7 +53,9 @@ export default Component.extend({
     if (sortInfo.descending) {
       return sessions.sortBy(sortInfo.column).reverse();
     }
-
+    if (isPresent(sessions) && this.preserveScroll.isListenerOff) {
+      this.scrollDown();
```

Comment:
Why here and not in `didInsertElement`? The sessions are passed in synchonsouly so other than rendering there shouldn't be a delay, could you add this instead `didInsertElement` and instead a `Ember.run.next` block?

Reply:
I vaguely recall in the previous versions of Ember, I was getting warnings in the console regarding setting a value during `didInsertElement`. They were recommending `didRender`. Do you recall?

Reply:
Oh wait, I called in the function because the lifecycle hooks would be called after page rendered but before session-list was rendered. So scroll down would occur before session-list were visible.